<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scanner</class>
 <widget class="QWidget" name="Scanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>314</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>234</width>
    <height>314</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>234</width>
    <height>314</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="resultt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="prefix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>prefix</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="suffix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>suffix</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pre">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="suf">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="mode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>78</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>keywedge</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mode_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>78</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>have Enter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Trigger</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Scanner</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
